skynet/modules/stt/vox/app.py

```diff
@@ -0,0 +1,72 @@
+import base64
```

Comment:
In my sample I used an alternativr which is much faster using SIMD instructions, maybe use that for some free perf?